spec/javascripts/actions/history_spec.js

```diff
@@ -0,0 +1,60 @@
+import * as actions from '../../../app/assets/javascripts/actions/story';
+import actionTypes from '../../../app/assets/javascripts/actions/actionTypes';
+import history from '../support/factories/historyFactory';
+import storie from '../support/factories/storyFactory';
```

Comment:
The singular of `stories` here would be `story` :smile: 
```suggestion
import story from '../support/factories/storyFactory';
```